packages/edit-post/src/components/visual-editor/style.scss

```diff
@@ -33,12 +33,6 @@
 	// color doesn't show through.
 	background-color: $white;
 	flex: 1;
-
-	cursor: text;
-
-	> * {
-		cursor: auto;
-	}
```

Comment:
We need to remove the cursor style as we can't apply it to only the bottom padding of an element.